pkg/converter/unstructured.go

```diff
@@ -30,6 +30,7 @@ type UnstructuredConverter struct {
 	o *unstructured.Unstructured
 }
 
+// ExtractObjectMeta creates an ObjectMeta object from an Unstructured object.
```

Comment:
I'm surprised this doesn't exist somewhere in K8s libraries.  Have you checked?

Reply:
I did check. I spent a long time looking at https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/unstructured/unstructured.go and neighboring directories, but couldn't find anything.